### PR TITLE
fix: added fully qualified type for justify modal footers

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Added Justify enum for fully qualified classnames and ease of import for the modal footer

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-771 #done #1386

### CODE REVIEW

File: packages/shared/src/components/utilities.tsx [hunk 1]
```diff
@@ -16,6 +16,12 @@ export enum Theme {
   Lettuce = 'lettuce',
 }
 
+export enum Justify {
+  End = 'justify-end',
+  Center = 'justify-center',
+  Between = 'justify-between',
+}
```
Comment (sshanzel) [inline packages/shared/src/components/utilities.tsx:23]: Love this enum. We should probably do this as well on some other CSS properties that we tend to clash with sometimes. Probably when we face it on the road.

**Review by sshanzel — APPROVED** [review]
Amazing cleanup you did there! Love the implementation. Hoping to see more of this in the future!

File: packages/shared/src/components/utilities.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/utilities.tsx:23]: Yep would be a great idea going forward, enums just make things more explicit and set in stone 🥳

Comment (el) [reply packages/shared/src/components/utilities.tsx:23]: I love this. We should use it on more utility classes (if they are passed as props)

**Comment by el** [comment]
@rebelchris Thanks for taking care of this. ❤️

**Pull request merged**